pkg/api/admin/openshiftcluster_convert.go

```diff
@@ -175,4 +183,13 @@ func (c *openShiftClusterConverter) ToInternal(_oc interface{}, out *api.OpenShi
 			Phase: api.InstallPhase(oc.Properties.Install.Phase),
 		}
 	}
+
```

Comment:
I dont think we need this in the ToInternal. We would not accept changes to this from admin

Reply:
I disagree.  We aim for 100% coverage on the converters in both directions; the validation will prevent any changes being made.